frontend/src/features/permits/components/dialog/ApplicationPendingPermitsModalDialog.tsx

```diff
@@ -0,0 +1,127 @@
+import { faExclamationTriangle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Dialog, Button, Typography } from "@mui/material";
+import "./ApplicationPendingPermitsModalDialog.scss";
+import { InfoBcGovBanner } from "../../../../common/components/banners/InfoBcGovBanner";
+import { MRT_ColumnDef, MaterialReactTable, useMaterialReactTable } from "material-react-table";
+import { NoRecordsFound } from "../../../../common/components/table/NoRecordsFound";
+import { useContext, useMemo } from "react";
+import { ApplicationListItem } from "../../types/application";
+import { ApplicationPendingPermitsModalColumnDefinition } from "./ApplicationPendingPermitsModalColumnDefinition";
+import { getDefaultNullableVal } from "../../../../common/helpers/util";
+import OnRouteBCContext from "../../../../common/authentication/OnRouteBCContext";
+import { defaultTableInitialStateOptions, defaultTableOptions } from "../../../../common/helpers/tableHelper";
+import { useApplicationsInProgressQuery } from "../../hooks/hooks";
+import { PPC_EMAIL, TOLL_FREE_NUMBER } from "../../../../common/constants/constants";
+
+/**
+ * Dynamically set the column
+ * @returns An array of column headers/accessor keys for Material React Table
+ */
+const getColumns = (): MRT_ColumnDef<ApplicationListItem>[] => {
+    return ApplicationPendingPermitsModalColumnDefinition();
+  };
+
+export const ApplicationPendingPermitsModalDialog = ({
+    showModal,
+    onCancel,
+  }: {
+    showModal: boolean;
+    onCancel: () => void;
+  }) => {
+    
+    const applicationPermitsPendingQuery = useApplicationsInProgressQuery({
+        pendingPermits: true,
+      });
+    
+    const { data } = applicationPermitsPendingQuery;
```

Comment:
This is a paginated API. We'll need handling for pagination.

Reply:
ok -- the initial design didn't include the pagination, but I'll run it past Girish given how the endpoint is structured...

Reply:
<img width="620" alt="image" src="https://github.com/bcgov/onroutebc/assets/84348052/7b0d91b3-8eb6-4ca6-86f1-6ac0aac455b7">